hydroflow_macro/src/lib.rs

```diff
@@ -59,32 +57,13 @@ fn root() -> proc_macro2::TokenStream {
     }
 }
 
-// May panic
-fn macro_invocation_path() -> PathBuf {
-    #[cfg(feature = "diagnostics")]
-    {
-        proc_macro::Span::call_site().source_file().path()
-    }
-    #[cfg(not(feature = "diagnostics"))]
-    {
-        std::env::current_dir().unwrap_or_else(|_| {
-            PathBuf::from(
-                std::env::var("CARGO_MANIFEST_DIR")
-                    .expect("Failed to determine fallback env var CARGO_MANIFEST_DIR."),
-            )
-        })
-    }
-}
```

Comment:
this part, the fallback really just doesn't do anything in terms of the relative path